doc/source/serve/multi-app.md

```diff
@@ -78,66 +81,80 @@ $ serve deploy config.yaml
 > Sent deploy request successfully!
```

Comment:
Is ray start needed?

Reply:
It's needed before running serve deploy. Do you mean if it's necessary to include in the docs?